src/main/java/com/exercism/junit/JUnitTestRunner.java

```diff
@@ -33,6 +33,7 @@ public void test(ClassLoader classLoader, Set<Path> classpathRoots) {
                 .selectors(DiscoverySelectors.selectClasspathRoots(classpathRoots))
                 .filters(ClassNameFilter.excludeClassNamePatterns(".+\\$.+"))
                 .configurationParameter("junit.platform.output.capture.stdout", "true")
+                .configurationParameter("junit.jupiter.params.displayname.default", "{displayName}[{index}] {argumentsWithNames}")
```

Comment:
I don't think we need the index in the default display name. I would change it to this:
```suggestion
                .configurationParameter("junit.jupiter.params.displayname.default", "{displayName} - {argumentsWithNames}")
```

Reply:
done